tests/python_functional/src/executors/command_executor.py

```diff
@@ -72,16 +67,11 @@ def run(self, command, stdout_path, stderr_path):
     def __process_std_outputs(self, stdout, stderr):
         stdout_content = stdout.open_file("r").read()
         stderr_content = stderr.open_file("r").read()
-        self.__logger.debug("Stdout:[{}]".format(stdout_content))
```

Comment:
These logs seem useful. Do we need to delete these? Maybe putting a lower loglevel?

Reply:
+1